src/AppBundle/Resources/views/Security/profile.html.twig

```diff
@@ -18,19 +18,19 @@
 
                   <ul class="list-group list-group-unbordered">
                     <li class="list-group-item text-left">
-                      <b>Login</b> <a class="pull-right">{{user.username}}</a>
+                      <b>{{ 'security.login.username'|trans }}</b> <a class="pull-right">{{user.username}}</a>
                     </li>
                     <li class="list-group-item text-left">
-                      <b>Last connection</b> <a class="pull-right">{{user.lastLogin|date('d/m/Y - H\\h i')}}</a>
+                      <b>{{'Last connection'|trans({}, 'kalamu')}}</b> <a class="pull-right">{{user.lastLogin|date('d/m/Y - H\\h i')}}</a>
```

Comment:
can you replace the date format with the `format_datetime` filter : `{{user.lastLogin|format_datetime}}`